poly-commit/src/lib.rs

```diff
@@ -156,8 +156,8 @@ pub trait PolynomialCommitment<F: PrimeField, P: Polynomial<F>, S: Cryptographic
     type VerifierKey: PCVerifierKey;
     /// The commitment to a polynomial.
     type Commitment: PCCommitment + Default;
-    /// The commitment randomness.
-    type Randomness: PCRandomness;
+    /// The state of commitment
```

Comment:
```suggestion
    /// Auxiliary commitment data that is output by the `commit` phase.
    /// It can be reused by the committer during the `open` phase.
    /// Not to be shared with the verifier.
```

Reply:
Wasn't "state" the name that we chose at the end?

Reply:
Oh yeah sure, the name can stay. I wanted to give more context though

Reply:
Changed the suggestion @autquis

Reply:
I would also say that this includes information like commitment randomness.